portfoliostyle.css

```diff
@@ -0,0 +1,438 @@
+	justify-content: space-evenly;
+	padding: 20px;
+	background-color: #808080;
+	color: #FFFFFF;
+	margin: 20px 0;
+	border-radius: 15px;
+}
+
+.project-item a {
+	padding: 10px;
+	background-color: #D3D3D3;
+	border-radius: 5px;
+	color: #000000;
+	border-left: 2px solid #6A5ACD;
+	border-top: 2px solid #6A5ACD;
+	text-decoration: none;
+}
+
+.project-item a:hover {
+	padding: 10px;
+	background-color: #000000;
+	border-radius: 5px;
+	color: #ADD8E6;
+	text-decoration: none;
+}
+
+.project-image {
+	max-width: 200px;
+	height: 150px;
+	margin: auto 0;
+	border: 2px solid #D3D3D3;
+	border-radius: 15px;
+}
+
+.project-info {
+	display: flex;
+	flex-direction: column;
+	width: 45%;
+}
+
+.project-info div {
+	display: flex;
+	justify-content: space-evenly;
+	margin: 10px 0;
+}
+
+.about {
+	padding: 20px 7vw;
+	background-color: #808080;
+	color: #FFFFFF;
+	margin: 20px 0;
+	border-radius: 15px;
+}
+
+.nav-item.cv {
+	color: #6A5ACD;
+	font-weight: bold;
+	background-color: #000000;
+	border-radius: 15px;
+}
+
+.nav-item.cv:hover {
+	color: #FFFFFF;
+	background-color: #ADD8E6;
+}
+
+/* social media buttons*/
+
+.fa {
+	font-size: 50px;
+	width: 100px;
+	height: 100px;
+	text-align: center;
+	text-decoration: none;
+	background-color: #000000;
+	color: #FFFFFF;
+	border-radius: 40%;
+}
+
+.media-links {
+	display: flex;
+	justify-content: space-evenly;
+	margin-bottom: 30px;
+}
+
+.media-links a {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	margin: 10px;
+	padding: 30px;
+}
+
+.media-links a:hover {
+	color: #ADD8E6;
+	border: 3px solid #FFFFFF;
+}
+
+/* timeline css*/
+
+.timeline {
+	position: relative;
+}
+
+.timeline::before {
+	content: "Hover over the below roles to learn more!";
+	color: #FFFFFF;
+	background-color: #000000;
+	opacity: 0.5;
+	padding: 5px;
+	border-radius: 5px;
+}
+
+.timeline::after {
+	content: "";
+	position: absolute;
+	width: 6px;
+	background-color: #FFFFFF;
+	top: 0;
+	bottom: 0;
+	left: 50%;
+	margin-left: -3px;
+}
+
+.container {
+	padding: 10px 40px;
+	position: relative;
+	width: 60%;
+}
+
+/* the balls on the timeline*/
+.container::after {
+	content: '';
+	position: absolute;
+	width: 25px;
+	height: 25px;
+	right: -17px;
+	background-color: #FFFFFF;
+	border: 4px solid #6A5ACD;
+	top: 15px;
+	border-radius: 50%;
+	z-index: 1;
+}
+
+.left {
+	left: -10%;
+}
+
+.right {
+	left: 50%;
+}
+
+.left::before {
+	content: " ";
+	height: 0;
+	position: absolute;
+	top: 22px;
+	width: 0;
+	z-index: 1;
+	right: 15px;
+	border: medium solid #FFFFFF;
+	border-width: 8px 0 8px 25px;
+	border-color: transparent transparent transparent #808080;
+}
+
+.right::before {
+	content: " ";
+	height: 0;
+	position: absolute;
+	top: 22px;
+	width: 0;
+	z-index: -1;
+	left: 15px;
+	border: medium solid #FFFFFF;
+	border-width: 8px 25px 8px 0;
+	border-color: transparent #808080
+ transparent transparent;
+}
+
+.right::after {
+	left: -16px;
+}
+
+.content {
+	padding: 20px 30px;
+	background-color: #808080
+;
```

Comment:
Semi-colon could be inline with the background-color for consistency

Reply:
Deleted :)